src/webgpu/api/validation/create_pipeline.ts

```diff
@@ -0,0 +1,15 @@
+export const description = `
```

Comment:
nit: This file is `create_pipeline` while the other one is `createPipeline`. What's the convention?

Reply:
camelCase is named after a function in the API, but createPipeline is not one (it's createRenderPipeline + createComputePipeline)